frontend/src/pages/ConnectionTest/ConnectionTest.tsx

```diff
@@ -729,7 +729,7 @@ function ReplaceConnection(props: {
  * @param props.srcObject video and audio source
  * @param props.ignoreAudio if true, audio tracks in `srcObject` will be ignored.
  */
-function Video(props: {
+export function Video(props: {
```

Comment:
We have a Video component in the atoms folder. if it is easy to adapt it, would be better to use that one